### ISSUE TITLE
[jfrog-platform] Custom volumes are ignored in subcharts

### ISSUE THREAD

**Issue by FalconerTC** [jfrog/charts#1413]
**Is this a request for help?**:

No
---

**Is this a BUG REPORT or FEATURE REQUEST?** (choose one):

BUG REPORT

**Version of Helm and Kubernetes**:
❯ helm version
version.BuildInfo{Version:"v3.5.2", GitCommit:"167aac70832d3a384f65f9745335e9fb40169dc2", GitTreeState:"dirty", GoVersion:"go1.15.7"}

Kubernetes 1.17.17-gke.2800

**Which chart**:
jfrog/jfrog-platform 0.4.1

**What happened**:
`artifactory-ha.artfactory.customVolumes` entries are ignored

**What you expected to happen**:
`artifactory-ha.artfactory.customVolumes` should be templated

**How to reproduce it** (as minimally and precisely as possible):
Define a value for `artifactory-ha.artfactory.customVolumes`

**Anything else we need to know**:
This looks to be because jfrog-platform defines a global `customVolumes` entry
```
  customVolumes: |
    {{ template "initdb-volume" . }}
```

Most charts use templates like the following, which when used with jfrog-platform means `Values.artifactory.customVolumes` will never template
```
{{- define "artifactory-ha.customVolumes" -}}
{{- if .Values.global.customVolumes -}}
{{- .Values.global.customVolumes -}}
{{- else if .Values.artifactory.customVolumes -}}
{{- .Values.artifactory.customVolumes -}}
{{- end -}}
{{- end -}}
```

**Comment by FalconerTC** [comment]
Work-around is to just empty the global customVolumes (as long as you don't actually need `initdb-volume`)
```
global:
  customVolumes: |
```

**Comment by chukka** [comment]
@FalconerTC  we already tracking  this  issue internally . Once fix is available, will  let you know (ETA - end of this quarter )

**Comment by geoff-carr-bzy** [comment]
I got hit by this issue too but with the `artifactory` chart setting the `customVolume:` in the `artifactory` values within the `jfrog-platform` chart. I worked around this issue by moving my custom volume to the global values but left the actual mounting in the `artifactory` values. In my case the `configMap` contains CA certificates so they could be needed on other pods too. As such looking forward to the fix for this issue.
